docs/whats-new/release-notes/v2_16_0.md

```diff
@@ -80,22 +72,24 @@ Zowe Version 2.3.0 contains the bug fixes that are described in the following to
 * Fixed configuration enabling JWT Token Refresh Functionality. ((#3474)[https://github.com/zowe/api-layer/issues/3474))
 * Fixed the Zowe logo and trademark info in the API Catalog. ((#3338)[https://github.com/zowe/api-layer/issues/3338])
 
-
-
-
 ### Zowe CLI
 
 #### Zowe CLI (Core)
 
-#### Zowe CLI Imperative Framework
-
-#### CICS Plug-in for Zowe CLI
+- Fixed the command `zowe daemon enable` installing an invalid daemon binary on macOS. ([#2126](https://github.com/zowe/zowe-cli/pull/2126))
+- Fixed error in the `zos-files list all-members` command that could occur when members contain control characters in the name. ([#2104](https://github.com/zowe/zowe-cli/pull/2104))
+- Resolved technical currency by updating the `tar` dependency. ([#2101](https://github.com/zowe/zowe-cli/issues/2101))
+- Resolved technical currency by updating the `markdown-it` dependency. ([#2106](https://github.com/zowe/zowe-cli/pull/2106))
+- Fixed default base profile missing in configuration generated by the `zowe config auto-init` command. ([#2084](https://github.com/zowe/zowe-cli/pull/2084))
+- Updated dependencies of the daemon client for technical currency. ([#2076](https://github.com/zowe/zowe-cli/pull/2076))
 
 #### DB2 Plug-in for Zowe CLI
 
-#### MQ Plug-in for Zowe CLI
+- Updated follow-redirects transitive dependency to resolve technical debt. ([#147](https://github.com/zowe/zowe-cli-db2-plugin/pull/147))
 
-#### IMS Plug-in for Zowe CLI
+#### FTP Plug-in for Zowe CLI
+
+- Fixed issue to return proper message for an active job. ([#2832](https://github.com/zowe/zowe-explorer-vscode/issues/2832))
```

Comment:
@adam-wolfe - Can we add more detail here? Also, is this PR link correct?

Reply:
Use this link: https://github.com/zowe/zowe-cli-ftp-plugin/issues/156

For a description, something like: Fixed error when listing spool for active jobs, now prints a warning that says that no spool files are available.